tests/JWT.Tests.Common/JWT.Tests.Common.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net6.0;netstandard2.0;net462</TargetFrameworks>
+    <TargetFrameworks>net7.0;net6.0;netstandard2.0;net462</TargetFrameworks>

```

Comment:
My main question would be why to do this? In other words, what's the benefit of targeting net7 if we don't call any net7 specific API?

Reply:
.net6 shouldn't have been needed too, as .netstandard2 should be enough. I'll check it out can be simplified

Reply:
The .NET Standard is dead, the recommendation by the .NET team is fo target certain "full" .NET version, such as 3.1 or 6. And that was my plan going forward - to add the LTS versions, i.e. skip 7.0 until really needed, snd later add 8.0

Reply:
Alright, fair enough. I'll try remove .net7